plugin_helpers/utils.py

```diff
@@ -48,3 +51,14 @@ def unique(seq):
 def rreplace(s, old, new, occurrence):
   li = s.rsplit(old, occurrence)
   return new.join(li)
+
+def quote(s):
+  if "windows" in platform.system().lower():
+    return _quote_windows_string(s)
+  else:
+    return shlex.quote(s)
+
+def _quote_windows_string(s):
+  if not s:
+    return '""'
```

Comment:
I don't think this special case is necessary. If `s` is falsey (e.g. an empty string), the line below will generate the same result (`'""'`).

Reply:
I pretty much stole that part from the shlex implementation.
I'll think that through once I am home.
I would however argue that if it is not an empty string but actually ```None``` calling the line below would result in an ```AttributeError```.

Reply:
I didn't think of `None` and assumed it's always a string. Makes sense 👍